client/transport/client.go

```diff
@@ -21,6 +21,8 @@ type Client interface {
 	Scheme() string
 	// TLSConfig returns any TLS configuration the client uses.
 	TLSConfig() *tls.Config
+
+	Close()
```

Comment:
thanks for your comment, i have just edit the code according to `io.Closer` interface.